src/interpreter-errors.ts

```diff
@@ -134,30 +154,56 @@ export class UnassignedVariable extends RuntimeSourceError {
 }
 
 export class InvalidNumberOfArguments extends RuntimeSourceError {
+  private calleeStr: string
+
   constructor(node: es.Node, private expected: number, private got: number) {
     super(node)
+    this.calleeStr = generate((node as es.CallExpression).callee)
   }
 
   public explain() {
-    return `Expected ${this.expected} arguments, but got ${this.got}`
+    return `Expected ${this.expected} arguments, but got ${this.got}.`
   }
 
   public elaborate() {
-    return 'TODO'
+    const calleeStr = this.calleeStr
+    const pluralS = this.expected === 1 ? '' : 's'
+
+    return `Try calling function ${calleeStr} again in the same way, but with ${
```

Comment:
I think 'in the same way' should be removed, since changing the number of arguments is not really 'the same way'.